enhancements/cloud-integration/infrastructure-external-platform-type.md

```diff
@@ -0,0 +1,560 @@
+---
+title: infrastructure-external-platform-type
+authors:
+  - "@lobziik"
+  - "@elmiko"
+reviewers:
+  - "@dhellmann"
+  - "@mhrivnak"
+  - "@rvanderp3"
+  - "@mtulio"
+  - "@deads2k, to review library-go, KCMO related parts andopenshift/api changes" 
+  - "@JoelSpeed, to review CCM, MAPO related parts and openshift/api changes"
+  - "@sinnykumari, to review MCO related parts"
+  - "@danwinship, to review CNO related parts"
+  - "@Miciah, to review Ingress related parts"
+  - "@openshift/openshift-team-windows-containers"
```

Comment:
@sinnykumari @danwinship @miciah , we are looking to round up final reviews on this. woud you mind taking a read of the parts that are specific to your concerns.

@aravindhp i'm not sure we have any specific windows concerns at the moment in here, but happy for your thoughts if we need to add more.

Reply:
Mostly lgtm. The main interaction with WMCO would be similar to MCO with it's treatment of `None` and `External` PlatformTypes.